### PR TITLE
Doc update: Shared classes cache layers

### PR DESCRIPTION
**Pull request by doveye**
Update -Xshareclasses topic and What's New, for the next step in the shrc layers feature ("createLayer" and "layer" suboptions).

https://github.com/eclipse/openj9-docs/issues/368

Signed-off-by: Esther Dovey doveye@uk.ibm.com

[skip ci]

### CODE REVIEW

**Comment by doveye** [comment]
Jenkins doc stage

**Comment by doveye** [comment]
Jenkins doc stage

**Comment by doveye** [comment]
Jenkins doc stage

**Comment by doveye** [comment]
Jenkins doc stage

**Comment by doveye** [comment]
Jenkins doc stage

**Comment by doveye** [comment]
Jenkins doc stage

**Comment by doveye** [comment]
Jenkins doc stage

File: docs/version0.17.md [hunk 1]
```diff
@@ -47,14 +48,20 @@ OpenJ9 release 0.17.0 supports OpenJDK 8, 11, and 13. Binaries are available fro
...
 
-Issue [#5611](https://github.com/eclipse/openj9/issues/5611) is fixed, so support for the Digest algorithm is re-enabled. For more information about this support, see [Cryptographic operations]( introduction.md#cryptographic-operations).
+A new suboption, [`destroyAllLayers`](xshareclasses.md#destroyalllayers), is available for destroying all the layers of a layered cache.
+
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** Because these suboptions are experimental, do not use them in a production environment.
+
+### New shared classes cache suboption to skip disk space check
+
+When creating a persistent shared classes cache, the OpenJ9 VM checks that there is sufficient disk space available on the file
+system. For file systems that do not support the checking of free space, you can set the `-Xshareclasses:noPersistentDiskSpaceCheck` option, which causes the VM to skip the space checking operation. If there isn't enough disk space available when the cache is written, a **SIGBUG** or **SIGSEGV** signal occurs and the VM ends. For more information, see the [-Xshareclasses:noPersistentDiskSpaceCheck](xshareclasses.md#nopersistentdiskspacecheck) option.
```
Comment (pshipton) [inline docs/version0.17.md]: This is going to revert the fix of SIGBUG -> SIGBUS

Comment (doveye) [reply docs/version0.17.md]: Well-spotted - bad merge conflict resolution :(  I have no idea how you undo such things, but it's only a typo so I've just changed it again in this PR.

**Pull request merged**